src/main/kotlin/com/exactpro/th2/codec/AbstractCodecProcessor.kt

```diff
@@ -61,7 +62,7 @@ abstract class AbstractCodecProcessor(
         return event.id
     }
 
-    protected fun Set<String>.onEachEvent(
+    protected fun Set<String?>.onEachEvent(
```

Comment:
I don't see any places where the set of IDs might have a null value. Are you sure that we need nullable type here?